wiremock/README.MD

```diff
@@ -134,48 +135,66 @@ WireMock can selectively proxy requests through to other hosts. Code snippets co
 #### Response Templating
 
 [WireMock](http://wiremock.org) supports by default [handlebars](http://handlebarsjs.com/) as templating engine.
-[Handlebars](http://handlebarsjs.com/) is not an OSGi-ready library and is not supported by AEM Stubs.
-
-Current implementation supports basic functionality of [Pebble](https://pebbletemplates.io/) templating engine.
+AEM Stubs by default has Global Templating Engine set for handlebars.
```

Comment:
AEM Stubs is using Handlebars template engine as \[global response transformer|link\] by default